patch/llpcPatchInOutImportExport.cpp

```diff
@@ -2797,7 +2803,8 @@ Value* PatchInOutImportExport::PatchGsBuiltInInputImport(
 Value* PatchInOutImportExport::PatchFsBuiltInInputImport(
     Type*        pInputTy,      // [in] Type of input value
     uint32_t     builtInId,     // ID of the built-in variable
-    Instruction* pInsertPos)    // [in] Where to insert the patch instruction
+    Instruction* pInsertPos,    // [in] Where to insert the patch instruction
+    Value*       pSampleId)     // [in] Sample ID; only needed for BuiltInSamplePosOffset
```

Comment:
Change the postion of the two parameters. Always make pInsertPos as the last parameter and try to void such default value parameter. If pSampleId is not used, give it nullptr.

We have learns many lessons from old driver codes where the default value parameters are abused everywhere. Everyone just wants to add his own parameter and does not want to change exisiting codes because of time-consuming manual work.